packages/core/src/components/Button/Button.styles.ts

```diff
@@ -21,8 +23,8 @@ export const StyledContentDiv = styled.div<{}>({
 });
 
 export const StyledIconSpan = styled.span<{}>({
-  marginRight: theme.button.marginIconRight,
```

Comment:
With these changes the buttons used on the bulk actions have more margin on the left.